.travis.yml

```diff
@@ -51,6 +55,10 @@ env:
     - LLVM_VERSION=3.4 SOLVERS=STP STP_VERSION=2.1.2 KLEE_UCLIBC=klee_uclibc_v1.0.0 DISABLE_ASSERTIONS=0 ENABLE_OPTIMIZED=0 COVERAGE=0 USE_CMAKE=1 USE_TCMALLOC=0 ASAN_BUILD=1
     - LLVM_VERSION=3.4 SOLVERS=STP STP_VERSION=2.1.2 KLEE_UCLIBC=klee_uclibc_v1.0.0 DISABLE_ASSERTIONS=0 ENABLE_OPTIMIZED=0 COVERAGE=0 USE_CMAKE=0 USE_TCMALLOC=0 ASAN_BUILD=1
 
+    # legacy build system
```

Comment:
I don't think we should have this many LLVM 3.5 and 3.6 configurations for now. Once we drop LLVM 2.9 we can re-evaluate this but to keep TravisCI testing time as short as possible I'd like to avoid adding too many configurations.